### ISSUE TITLE
docker: Error response from daemon: could not select device driver "" with capabilities: [[gpu]].

### ISSUE THREAD

**Issue by jurassicjordan** [SortAnon/ControllableTalkNet#31]
docker: Error response from daemon: could not select device driver "" with capabilities: [[gpu]].

did I set docker up wrong?

**Comment by pmanavado01** [comment]
No, docker just needs nvidia container toolkit to be installed correctly. 
If you are on ubuntu or a similar distro using an nvidia card then run these commands:


distribution=$(. /etc/os-release;echo $ID$VERSION_ID)       && curl -s -L https://nvidia.github.io/libnvidia-container/gpgkey | sudo apt-key add -       && curl -s -L https://nvidia.github.io/libnvidia-container/$distribution/libnvidia-container.list | sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list

sudo apt-get update

sudo apt-get install -y nvidia-docker2

sudo systemctl restart docker

sudo docker run --shm-size 2GB -it --gpus all docurdt/heal 

sudo apt-get update

sudo systemctl restart docker



It should then be able to recognize the card.

**Comment by GhostDog98** [comment]
For people on arch linux, I ran just had to run these commands:
```bash
yay nvidia-container-toolkit
sudo systemctl restart docker
```
